services/pane.js

```diff
@@ -55,21 +57,58 @@ function open(header, innerEl) {
 }
 
 /**
+ * get today's date, in YYYY-MM-DD
+ * @returns {string}
+ */
+function today() {
+  return moment().format('YYYY-MM-DD');
+}
+
+/**
+ * get current time, in HH:mm:ss
+ * @returns {string}
+ */
+function now() {
+  return moment().format('HH:mm');
+}
+
+/**
  * open publish pane
+ * @returns {Promise}
  */
 function openPublish() {
   var header = 'Schedule Publish',
-    actionsEl = dom.create(`
-      <div class="actions">
-        <button class="publish-now">Publish Now</button>
-      </div>
-    `), // todo: add other publishing actions, depending on page state
-    // e.g. schedule publish, message, cancel schedule, unpublish
-    el = open(header, actionsEl);
-
-  // init controller for publish pane
-  ds.controller('publish-pane', publishPaneController);
-  ds.get('publish-pane', el.querySelector('.actions'));
+    actions = dom.create(`<div class="actions"></div>`),
+    schedule = dom.create(`<form class="schedule">
+      <input class="schedule-input" type="date" min="${today()}" value="${today()}"></input>
```

Comment:
Why call today() multiple times? Why not eliminate the now() and today() functions and just replace them with vars at the top of the function?
var now = moment().format('HH:mm'),
    today = moment().format('YYYY-MM-DD');